ganttproject/src/biz/ganttproject/impex/csv/GanttCSVExport.java

```diff
@@ -62,13 +60,30 @@
  * @author athomas
  */
 public class GanttCSVExport {
+
+  public enum FileExtensionEnum {
```

Comment:
Finally, let's admit: it is not an enumeration of file extensions, it is enumeration of export formats. Please rename it to `Format`